hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java

```diff
@@ -3891,6 +3891,8 @@ private void processMisReplicatesAsync() throws InterruptedException {
         // initialisation, then progress might be different.
         reconstructionQueuesInitProgress = Math.min((double) totalProcessed
             / totalBlocks, 1.0);
+        LOG.info("Reconstruction queues initialisation progress: {}, total number of blocks " +
```

Comment:
I have always believed that it is not a good practice to involve some IOs(include print log) inner Read/Write Lock, here in global write lock. Please check if it could move out.

Reply:
HI @Hexiaoqiao Sir , thanks help me review it . 
Your suggestion is right, I checked this, the print log can be moved out of the lock, I will update the PR later.